src/common/helpers/formatPriceRequestRounds.ts

```diff
@@ -0,0 +1,146 @@
+import { BigNumber, utils } from "ethers";
+import { PriceRequestRound } from "common/hooks/useVoteData";
+import formatRequestKey from "./formatRequestKey";
+import toWei from "common/utils/convertToWeiSafely";
+
+export interface FormattedPriceRequestRound {
+  [key: string]: {
+    totalSupplyAtSnapshot: string;
+    uniqueCommits: string;
+    revealedVotes: string;
+    revealedVotesPct: string;
+    uniqueReveals: string;
+    uniqueRevealsPctOfCommits: string;
+    correctVotes: string;
+    correctlyRevealedVotesPct: string;
+    roundInflationRate: string;
+    roundInflationRewardsAvailable: string;
+    rewardsClaimed: string;
+    rewardsClaimedPct: string;
+    uniqueClaimers: string;
+    uniqueClaimersPctOfReveals: string;
+  };
+}
+
+const fromWei = utils.formatUnits;
+
+export default function formatPriceRequestRounds(
```

Comment:
might want to note wherever you have pulled code and refactored it directly from old dapp. will help people when reviewing to know its not totally new code